actionpack/lib/action_dispatch/railtie.rb

```diff
@@ -37,7 +37,7 @@ class Railtie < Rails::Railtie # :nodoc:
       ActionDispatch::ExceptionWrapper.rescue_responses.merge!(config.action_dispatch.rescue_responses)
       ActionDispatch::ExceptionWrapper.rescue_templates.merge!(config.action_dispatch.rescue_templates)
 
-      config.action_dispatch.authenticated_encrypted_cookie_salt = "authenticated encrypted cookie" if config.action_dispatch.use_authenticated_cookie_encryption
+      ActionDispatch::CookieSecurity.configure! config.action_dispatch
```

Comment:
This is a normal hash, so how `config.action_dispatch["action_dispatch.secret_key_base"]` would return something?